### ISSUE TITLE
Fix Low-Contrast Grey in Diagrams

### ISSUE THREAD

**Issue by tajmone** [tajmone/Git-Buch_EN#17]
The grey colour of the tex in the `D'` box in Diagram `svn-dcommit.svg` doesn't meet the WebAIM contrast accessibility criteria.

Try to find a solution (another grey, or bold, italic, etc.).

See [comments on Issue #15](https://github.com/tajmone/Git-Buch_EN/issues/15#issuecomment-816252635) for the details.

**Comment by tajmone** [comment]
@SicroAtGit, in order to keep the colors accessible to the visually impaired, I've colored the text in black (like the rest) and used a dotted border on the box instead, which should be interpreted as having the same meaning.

Let me know if the diagrams looks better now:

![svn-dcommit.svg](https://raw.githubusercontent.com/tajmone/Git-Buch_EN/alpha-dev/docs_src/images/svn-dcommit.svg)

**Comment by SicroAtGit** [comment]
Good work.
